safety_limiter/src/safety_limiter.cpp

```diff
@@ -641,28 +684,31 @@ class SafetyLimiterNode
   }
   void cbCloud(const sensor_msgs::PointCloud2::ConstPtr& msg)
   {
-    pcl::PointCloud<pcl::PointXYZ>::Ptr pc(new pcl::PointCloud<pcl::PointXYZ>());
+    sensor_msgs::PointCloud2 cloud_msg_fixed;
 
     try
     {
-      sensor_msgs::PointCloud2 pc2_tmp;
-      geometry_msgs::TransformStamped trans = tfbuf_.lookupTransform(
-          frame_id_, msg->header.frame_id, msg->header.stamp, ros::Duration(0.1));
-      tf2::doTransform(*msg, pc2_tmp, trans);
-      pcl::fromROSMsg(pc2_tmp, *pc);
+      geometry_msgs::TransformStamped cloud_to_fixed = tfbuf_.lookupTransform(
+          fixed_frame_id_, msg->header.frame_id, msg->header.stamp, ros::Duration(0.5 / hz_));
```

Comment:
please make this same as https://github.com/at-wat/neonavigation/pull/421/files#diff-c9e590a8e91490fdc29a9271015f9508R346-R357

Reply:
done